### ISSUE TITLE
Static kickstarter update replaces modified netdata.conf in /etc

### ISSUE THREAD

**Issue by kromit** [netdata/netdata#9111]
##### Bug report summary

After an automatic update `netdata.conf` was replaced by default empty file, opening previously closed port.

##### OS / Environment

```/etc/centos-release:CentOS Linux release 7.7.1908 (Core)
/etc/os-release:NAME="CentOS Linux"
/etc/os-release:VERSION="7 (Core)"
/etc/os-release:ID="centos"
/etc/os-release:ID_LIKE="rhel fedora"
/etc/os-release:VERSION_ID="7"
/etc/os-release:PRETTY_NAME="CentOS Linux 7 (Core)"
/etc/os-release:ANSI_COLOR="0;31"
/etc/os-release:CPE_NAME="cpe:/o:centos:centos:7"
/etc/os-release:HOME_URL="https://www.centos.org/"
/etc/os-release:BUG_REPORT_URL="https://bugs.centos.org/"
/etc/os-release:
/etc/os-release:CENTOS_MANTISBT_PROJECT="CentOS-7"
/etc/os-release:CENTOS_MANTISBT_PROJECT_VERSION="7"
/etc/os-release:REDHAT_SUPPORT_PRODUCT="centos"
/etc/os-release:REDHAT_SUPPORT_PRODUCT_VERSION="7"
/etc/os-release:
/etc/redhat-release:CentOS Linux release 7.7.1908 (Core)
/etc/system-release:CentOS Linux release 7.7.1908 (Core)

```

##### Netdata version

 v1.22.1-74-gd218786a

##### Component Name

authomatic update

##### Steps To Reproduce

1. install netdata using static kikstarter
2. change `/opt/netdata/etc/netdata/netdata.conf`
3. wait for netdata update
4. `/opt/netdata/etc/netdata/netdata.conf` is replaced with the default file

##### Expected behavior

never enable automatic update by default.
never replace configuration files.

**Comment by prologic** [comment]
I believe this was fixed with #9046 and a duplicate of #8531 and #8679 -- The fix should be in the latest nightly. Please if this is not the case feel free to reopen this issue or comment.

**Comment by kromit** [comment]
Since I've had to re-install a stable release on 10+ nodes and disable all updates entirely, I am sure, I will never enable it again. So you have to wait until it deletes some production config of someone else. 😉 cheers
